docs/HowToGuides/GettingStarted.md

```diff
@@ -272,6 +277,10 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
    - If the build fails:
      See [Troubleshooting build issues](#troubleshooting-build-issues).
 
+   If you would like to build the Swift corelibs, ie swift-corelibs-libdispatch,
+   swift-corelibs-foundation, and swift-corelibs-xctest, also on Linux, add the
+   `--xctest` flag to `build-script`.
```

Comment:
```suggestion
   If you would like to additionally build swift-corelibs-libdispatch,
   swift-corelibs-foundation, and swift-corelibs-xctest (these are particularly
   useful on Linux) add the `--xctest` flag to `build-script`.
```

Reply:
Reworded a bit but [swift-corelibs-foundation](https://github.com/apple/swift/blob/main/utils/build-script-impl#L2317) and [swift-corelibs-libdispatch](https://github.com/apple/swift/blob/main/utils/build-script-impl#L2382) are not built on macOS, so kept the emphasis on this flag only being for Linux.